src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -85,6 +86,7 @@ public void Default(EnvironmentVariableTarget target)
 
 
         [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "SetEnvironmentVariableW isn't supported in Appx.")] // Issue #19870
```

Comment:
Yes we can do that. Either you or me. I can't do that before mid next week (when serialization is hopefully finished).